tenable/io/v3/assets/api.py

```diff
@@ -0,0 +1,403 @@
+        ``netbios_name``, ``mac_address``.  Each record may also contain
+        additional properties.
+
+        Args:
+            *assets:
+                One or more asset definition dictionaries
+            source:
+                An identifier to be used to upload the assets.
+
+        Returns:
+            str:
+                The job UUID.
+
+        Examples:
+            import single asset:
+
+            >>> tio.v3.assets.asset_import('example_source', {
+            ...     'fqdn': ['example.py.test'],
+            ...     'ipv4': ['192.168.254.1'],
+            ...     'netbios_name': 'example',
+            ...     'mac_address': ['00:00:00:00:00:00']
+            ... })
+
+            import multiple asset:
+
+            >>> tio.v3.assets.asset_import('multiple_asset_example_source',
+            ...     {
+            ...         'fqdn': ['example_one.py.test'],
+            ...         'ipv4': ['192.168.1.1'],
+            ...         'netbios_name': 'example_one',
+            ...         'mac_address': ['00:00:00:00:00:00']
+            ...     },{
+            ...         'fqdn': ['example_two.py.test'],
+            ...         'ipv4': ['192.168.255.1'],
+            ...         'netbios_name': 'example_two',
+            ...         'mac_address': ['00:00:00:00:00:00']
+            ...     })
+        '''
+        # We will likely want to perform some more stringent checking of the
+        # asset resources that are being defined, however a simple type check
+        # should suffice for now.
+        schema = ImportAssetSchema()
+        payload = schema.dump(schema.load({'assets': assets,
+                                           'source': source}))
+        return self._post('import', json=payload)['asset_import_job_uuid']
+
+    def list_import_jobs(self) -> List:
+        '''
+        Returns a list of asset import jobs.
+
+        :devportal:`assets: list-import-jobs <assets-list-import-jobs>`
+
+        Returns:
+            List:
+                List of job records.
+
+        Examples:
+            >>> for job in tio.v3.assets.list_import_jobs():
+            ...     pprint(job)
+        '''
+        return self._get('import/jobs')['asset_import_jobs']
+
+    def import_job_details(self, uuid: UUID) -> Dict:
+        '''
+        Returns the details about a specific asset import job.
+
+        :devportal:`assets: import-job-info <assets-import-job-info>`
+
+        uuid:
+            The UUID (unique identifier) for the job.
+
+        Returns:
+            Dict:
+                The job Resource record.
+
+        Examples:
+            >>> job = tio.v3.assets.import_job_details(
+            ...     '00000000-0000-0000-0000-000000000000')
+            >>> pprint(job)
+        '''
+        return self._get(f'import/jobs/{uuid}')
+
+    def move_assets(
+            self, source: UUID, destination: UUID, targets: List[str]
+    ) -> int:
```

Comment:
currently is an anti-pattern for python.

```suggestion
    def move_assets(self, 
                    source: UUID, 
                    destination: UUID, 
                    targets: List[str]
                    ) -> int:
```